x/profile/client/cli/tx_create_coordinator.go

```diff
@@ -0,0 +1,66 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/profile/types"
+)
+
+var _ = strconv.Itoa(0)
+
+const (
+	flagIdentity = "identity"
+	flagWebsite  = "website"
+	flagDetails  = "details"
+)
+
+func CmdCreateCoordinator() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-coordinator",
+		Short: "Broadcast message create-coordinator",
```

Comment:
```suggestion
		Short: "Create a new coordinator profile",
```

`message` command actually has a `--desc` flag to precise a description of the message other than the default generated one